src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Unix.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+
+namespace System.Diagnostics.Tracing
+{
+    internal sealed class RuntimeEventSourceHelper
+    {
+        internal static Interop.Sys.ProcessCpuInformation cpuInfo;
```

Comment:
>internal static Interop.Sys.ProcessCpuInformation cpuInfo; [](start = 8, length = 58)

Is this used anywhere else? Can it be a local variable inside `GetCpuUsage` instead? Or make it `private`?

Reply:
GetCpuUtilization needs it to keep track of previously recorded user/kernel/system times so it can't be a local variable. I'll make it private.